packages/mask/src/plugins/Tips/types/tip.ts

```diff
@@ -3,10 +3,15 @@ export enum TipType {
     NFT = 'nft',
 }
 
+export interface AddressConfig {
```

Comment:
Only a bit of advice.

```suggestion
export interface TipAccount {
```

Reply:
Good suggestion. Changed.